src/replay/execution.zig

```diff
@@ -89,14 +91,53 @@ pub fn replayActiveSlots(state: ReplayExecutionState) !bool {
         slot_statuses.deinit(state.allocator);
     }
     for (active_slots) |slot| {
-        state.logger.info().logf("replaying slot: {}", .{slot});
+        state.logger.debug().logf("replaying slot: {}", .{slot});
         const result = try replaySlot(state, slot);
         errdefer result.deinit(state.allocator);
         try slot_statuses.append(state.allocator, .{ slot, result });
     }
     return try processReplayResults(state, slot_statuses.items);
 }
 
+pub const LogHelper = struct {
```

Comment:
Maybe no need to make it public?

```suggestion
const LogHelper = struct {
```

Looks very specific to replay execution, it may not be applicable anywhere else?

Reply:
it is in ReplayState so it needs to be public.